internal/commands/config/list_trusted_builders_test.go

```diff
@@ -1,19 +1,20 @@
-package commands_test
+package config_test
 
 import (
 	"bytes"
 	"io/ioutil"
 	"os"
 	"testing"
 
+	cmdConfig "github.com/buildpacks/pack/internal/commands/config"
+
 	"github.com/buildpacks/pack/internal/config"
```

Comment:
Should be grouped with 3rd import group. May want to check IDE settings. Also, I believe we have some automation around this via `make lint` but it may not be working here... 🤔